keps/NNNN-kep-template/README.md

```diff
@@ -452,6 +453,36 @@ You can take a look at one potential example of such test in:
 https://github.com/kubernetes/kubernetes/pull/97058/files#diff-7826f7adbc1996a05ab52e3f5f02429e94b68ce6bce0dc534d1be636154fded3R246-R282
 -->
 
+### Testing Quality
+
+<!--
+This section must be completed when targeting alpha to a release.
+-->
+
+###### What is the current (unit) test coverage of packages that will be touched when implementing this KEP?
+
+<!--
+List the packages where non-boilerplate code will be added/changed with their
+current coverage as reported by test-coverage suite under
+https://testgrid.k8s.io/sig-testing-canaries#ci-kubernetes-coverage-unit
+-->
+
+###### If the coverage is below 80%, what additional tests will be added before implementing this KEP?
+
+<!--
+We need to ensure that all existing areas of the code that will be touched by
+this enhancement are adequatly tested to mitigate the risk of regressions.
+--->
+
```

Comment:
I would add a question re: e2e specifically asking if an existing test package already exists where tests would be added vs a new test package needs to be created. Along with clarifying whether the sig has adequate support/capacity to create and maintain these new tests.

Reply:
Will add.